spring-cloud-vault-config/src/main/java/org/springframework/cloud/vault/config/ClientAuthenticationFactory.java

```diff
@@ -102,42 +104,42 @@ ClientAuthentication createClientAuthentication() {
 
 		switch (this.vaultProperties.getAuthentication()) {
 
-		case APPID:
-			return appIdAuthentication(this.vaultProperties);
+			case APPID:
```

Comment:
Can you please run the Maven build to remove formatting changes?

Reply:
I think that it's been resolved. Apologies for taking so long.

Reply:
Not sure if you're waiting for me to resolve this, but I think it's resolved.